docs/reference/esql/esql-across-clusters.asciidoc

```diff
@@ -19,6 +20,70 @@ include::{es-ref-dir}/search/search-your-data/search-across-clusters.asciidoc[ta
 
 include::{es-ref-dir}/search/search-your-data/search-across-clusters.asciidoc[tag=ccs-proxy-mode]
 
+[discrete]
+[[esql-ccs-security-model]]
+==== Security model
+
+{es} supports two security models for cross-cluster search (CCS):
+
+* <<esql-ccs-security-model-certificate, TLS certificate authentication>>
+* <<esql-ccs-security-model-api-key, API key authentication>>
+
+[discrete]
+[[esql-ccs-security-model-certificate]]
+===== TLS certificate authentication
+
+TLS certificate authentication makes sense in single administrator scenarios where you have full control over both clusters.
+Importantly, you must ensure that named roles and their privileges are identical in both clusters.
+Refer to <<remote-clusters-cert, TLS certificate authentication>> for prerequisites and detailed setup instructions.
+
+[discrete]
+[[esql-ccs-security-model-api-key]]
+===== API key authentication
+
+[NOTE]
+====
+`ENRICH` is *not supported* in this version when using {esql} with the API key based security model.
+====
+
+The following information pertains to using {esql} across clusters with the <<remote-clusters-api-key, *API key based security model*>>.
+API key based cross-cluster search (CCS) enables more granular control over allowed actions between clusters.
+As a result, defining roles and privileges is more complex than with the certificate based security model.
+For example, it makes sense in scenarios when you have different administrators for different clusters.
+
+You will need to create an API key using the <<security-api-create-cross-cluster-api-key,Create cross-cluster API key>> API or using the {kibana-ref}/api-keys.html[Kibana API keys UI].
+
+The following example API call creates a role that can query remote indices using {esql} with the API key based security model.
+
+[source,console]
+----
+POST /_security/role/remote1
+{
+  "cluster": ["cross_cluster_search"], <2>
```

Comment:
nit: can these be re-ordered to read top down 1-5 ?

Reply:
totally was trying to give a sense of relative importance but agree might be a net bad reading experience